src/main/kotlin/com/google/protobuf/gradle/KotlinInteroperability.kt

```diff
@@ -0,0 +1,152 @@
+package com.google.protobuf.gradle
+
+import org.gradle.api.NamedDomainObjectContainer
+import org.gradle.api.Project
+import org.gradle.api.artifacts.*
+import org.gradle.api.artifacts.dsl.DependencyHandler
+import org.gradle.kotlin.dsl.*
+
+
+fun Project.protobuf(configuration: ProtobufConfigurator.() -> Unit) {
+    project.convention.getPlugin(ProtobufConvention::class.java).protobuf.apply(configuration)
+}
+
+fun ProtobufConfigurator.protoc(closure: ExecutableLocator.() -> Unit) {
+    protoc(delegateClosureOf(closure))
```

Comment:
I am still trying to wrap my head around `delegateClosureOf` vs `closureOf` (https://github.com/gradle/kotlin-dsl/blob/master/doc/getting-started/Closures.md). Is `delegateClosureOf` meant for closures that make use of the [closure delegate from groovy](http://groovy-lang.org/closures.html#closure-owner) ? As far as I can tell this plugin does not use this feature.

Reply:
So is a by-product of my ignorance of groovy. I updated to use ```closureOf```